models/civicrm_sync_settings.py

```diff
@@ -2,9 +2,88 @@
 from odoo import fields, models
 
 
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    custom_invoice_reference_prefix = fields.Char(
```

Comment:
Is this a duplicate of L48?

Reply:
No, it is not dublicate, the reason of this code is possibility to save different settings for different companies.

Reply:
but do we really need that ? @guanhuan

Reply:
Interesting. It can be useful in the future. Thanks for explaining @naulin. Let's keep that.